manifests/init.pp

```diff
@@ -263,6 +263,9 @@
       servername    => $foreman::servername,
     }
 
+    include pulpcore::plugin::file
```

Comment:
Could you respect the flags?
```suggestion
    if $enable_file {
      include pulpcore::plugin::file
    }
```

Reply:
Sure, I often forget about those things since we deploy and expect a more opinionated deployment with Katello.